frontend/components/Course/CourseWrapper.tsx

```diff
@@ -1,4 +1,5 @@
-import React, { useContext, MutableRefObject } from "react";
+import { useContext, MutableRefObject } from "react";
+import * as React from "react";
```

Comment:
Looked at this component, I don't think we need this import anymore

Commenting so I remember to look at the toggleNotifs function again

Reply:
This seems to be used in toggleNotifs, but the parameters are untouched so I removed them - good point!